### PR TITLE
Error logs --follow if events-backend != journald, event-logger=journald

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://github.com/containers/podman/issues/11255

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: libpod/container_log_linux.go [hunk 1]
```diff
@@ -44,6 +44,9 @@ func (c *Container) initializeJournal(ctx context.Context) error {
 }
 
 func (c *Container) readFromJournal(ctx context.Context, options *logs.LogOptions, logChannel chan *logs.LogLine) error {
+	if options.Follow && c.runtime.EventerType() != "journald" {
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply libpod/container_log_linux.go]: Moved that check up to here.

**Comment by rhatdan** [comment]
The logs error is not going to be fixed in the issue.

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM
but the CI isn't buying them.

**Comment by umohnani8** [comment]
LGTM

**Pull request merged**